src/ngraph/op/experimental/quantized_conv_bias.hpp

```diff
@@ -40,7 +40,8 @@ namespace ngraph
                                      const CoordinateDiff& padding_above,
                                      const Strides& data_dilation_strides,
                                      const std::shared_ptr<Node> scale,
-                                     const bool with_relu = false);
+                                     const bool with_relu = false,
+                                     const std::shared_ptr<Node> output_scale = nullptr);
```

Comment:
Won't this simply crash if called with `output_scale==nullptr`? (The constructor calls `check_single_output_args` on a vector including `output_scale`, which should deref the null pointer.) If so, we should remove the default value.